group_vars/rabbitmq/public.yml

```diff
@@ -6,4 +6,4 @@ SANITY_CHECK_LIVE_PORTS:
     port: 15671
     message: "RabbitMQ management does not respond on port 15671."
 
-TARSNAP_BACKUP_FOLDERS: /etc/rabbitmq /var/lib/rabbitmq /etc/letsencrypt
+TARSNAP_BACKUP_FOLDERS: /usr/local/lib/rabbitmq/backup /etc/rabbitmq /etc/letsencrypt
```

Comment:
I'm surprised that the scoping works out this way. I thought that it wouldn't due to stuff like `RABBITMQ_EXPORT_DIR` only being defined in the `rabbitmq` role and `TARSNAP_BACKUP_FOLDERS` being used in the tarsnap backup role, but it works. Thanks!